src/__tests__/actions/tagsActions.test.js

```diff
@@ -0,0 +1,52 @@
+import configureStore from 'redux-mock-store'
+import * as tagsActions from '../../redux/actions/tagsActions'
+import { ADD_TAG, EDIT_TAG, DELETE_TAG, } from '../../constants'
+
+const mockStore = configureStore()
+const store = mockStore()
+
+describe('tagsActions', () => {
+  const prevName = 'myTag'
+  const newName = 'MyTAG'
+  const tagName = 'myTag'
+
+  beforeEach(() => {
+    store.clearActions()
+  })
+
+  test('addTag action creator have to send an expected action', () => {
```

Comment:
...creator _HAS_...

Reply:
Maybe 'the store should dispatch addTag action'?

Reply:
agree